msm/src/serialization/mod.rs

```diff
@@ -47,23 +47,34 @@ mod tests {
     #[test]
     fn test_completeness() {
         let mut rng = o1_utils::tests::make_test_rng();
-        const DOMAIN_SIZE: usize = 1 << 5;
+        // Must be at least 1 << 15 to support rangecheck15
```

Comment:
This will slow things quite a bit... either we need cached SRS (preferred by me), or we could make the gate generic and run tests with `LIMB_SIZE = 7` for example.